toonz/sources/toonz/xsheetviewer.cpp

```diff
@@ -694,18 +835,12 @@ void XsheetViewer::paintEvent(QPaintEvent*)
 //-----------------------------------------------------------------------------
 
 void XsheetViewer::resizeEvent(QResizeEvent *event) {
-  int w              = width();
-  int h              = height();
-  int scrollBarWidth = 16;
-  m_noteScrollArea->setGeometry(3, 1, m_x0 - 4, m_y0 - 3);
-  m_cellScrollArea->setGeometry(m_x0, m_y0, w - m_x0, h - m_y0);
-  m_columnScrollArea->setGeometry(m_x0, 1, w - m_x0 - scrollBarWidth, m_y0 - 3);
```

Comment:
Fixed. Went through and compared the look from the current version and made sure this version looked the same or similar.  NOTE: I think a merge from another PR caused a number of font size changes.